src/util/mock.js

```diff
@@ -29,3 +29,45 @@ export const Projects = [{
   available: false,
   fbLink: '//facebook.com',
 }];
+
+export const Members = [{
+  id: 1,
+  name: 'Василій',
+  surname: 'Іванов',
+  nickName: 'Васив',
+  avatar: 'http://lumpics.ru/wp-content/uploads/2017/11/Programmyi-dlya-sozdaniya-avatarok.png',
+  degree: {
```

Comment:
It is better to store degrees as an array (we do have lots of members in SB, who has Bachelor's & Master's degrees both)